website/pages/use-cases/network-infrastructure-automation.jsx

```diff
@@ -37,13 +77,13 @@ export default function NetworkInfrastructureAutomationPage() {
           links: [
             {
               text: 'Read More',
-              url: '/docs/partnerships',
+              url: '/docs/nia-integrations',
```

Comment:
This 404's, I think you mean this:

```suggestion
              url: '/docs/integrate/nia-integration',
```

Reply:
Just committing this one here; this appears to be the only logical page you could have intended!